### DISCUSSION TITLE
What3words

### DISCUSSION THREAD

**Discussion by Blacky269** [osmandapp/OsmAnd#13376] [Ideas]
Hi guys, just wondering if you have plans or it is possible to add What3words location to the Osmand app?

https://what3words.com/about

Thanks in advance
Brett

**Comment by mariush444** [comment]
probably the plugin can be added
API is here
https://developer.what3words.com/public-api/playground

code for implementation for android is here
https://developer.what3words.com/tutorial/creating-an-android-autosuggest-edittext-box-with-what3words-autosuggest-component
but I don't think that this feature is widely needed

**Reply by Blacky269** [reply]
In Australia, the emergency services use What3words as a means of pinpointing accidents, fires etc so it would be very handy to be able to do it on OsmAnd.

**Comment by stephan75** [comment]
Maybe see https://wiki.openstreetmap.org/wiki/What3words for some background information ...
